web_src/js/features/repo-release.js

```diff
@@ -41,6 +42,10 @@ function initTagNameEditor() {
       showElem('#tag-target-selector');
       tagHelper.textContent = value ? newTagHelperText : defaultTagHelperText;
     }
+  };
+  hideTargetInput(tagNameInput); // update on page load because the input may have a value
+  tagNameInput.addEventListener('change', (e) => {
```

Comment:
```suggestion
  tagNameInput.addEventListener('input', (e) => {
```

Use "input" event?

Reply:
But if user pastes content, it will not be triggered.

Reply:
What do you mean ? "input" not be triggered or "change" not be triggered?

Reply:
`input` will not be triggered but `change` will when pasting text on the input box.

Reply:
Are you sure "input" won't be triggered?

Reply:
e6ac5ed9a0, OK. Looks like `input` does also work.

Reply:
"input" and "change" are different.